fecgov/fec-cms
# [DON'T MERGE] 5641 - 2024 presidential map

## Summary

- Resolves #5641 

Updated the presidential finance map to get ready for 2024 and include previous years (if API will support it)

### Required reviewers

- 1 front-end
- 1 UX
- 1 back-end/db?
- 1 product owner?

## Impacted areas of the application

Only the presidential map is affected

## Screenshots

![image](https://user-images.githubusercontent.com/26720877/223772268-cfbae1df-cff6-4b8a-9198-a5170cb65433.png)

![image](https://user-images.githubusercontent.com/26720877/223772645-b5bb29be-1903-4700-948c-563018c3d54c.png)


## Related PRs

None

## How to test

- Pull the branch
- npm i
- npm run build
- Check the [presidential map](http://127.0.0.1:8000/data/candidates/president/presidential-map/)
- **Known issues**
   - At this point, 2024 data doesn't exist so
   - the map default to 2020.
   - At this point, data before 2016 doesn't exist (not sure whether it will)
